lib/net/imap.rb

```diff
@@ -786,6 +799,15 @@ def expunge
       end
     end
 
+    # Similar to #expunge, but takes a set of unique identifiers as
+    # argument.
+    def uid_expunge(set)
```

Comment:
No multi-line suggestions, yet!  😉

```suggestion
    def uid_expunge(uid_set)
```